Meta/download_file.py

```diff
@@ -74,7 +74,7 @@ def main():
 
     os.makedirs(output_file.parent, exist_ok=True)
 
-    request = urllib.request.Request(args.url, headers={"Accept-Encoding": "gzip"})
+    request = urllib.request.Request(args.url, headers={"Accept-Encoding": "gzip", "User-Agent": "Mozilla/5.0"})
```

Comment:
Does it have to be Moz/5.0, or does (e.g.) "serenityos Meta/download_file.py" work as well?

Thanks for the PR!